decidim-core/lib/decidim/query_extensions.rb

```diff
@@ -15,7 +15,7 @@ def self.extend!(type)
           type !types[ProcessType]
           description "Lists all processes."
 
-          resolve ->(_obj, _args, ctx) {
+          resolve lambda { |_obj, _args, ctx|
```

Comment:
Hehe, so there was a bug in coverage tracking. I fixed it and codecov got green and global coverage increased by 2%! However, CircleCI took ages to finish, not sure why. I'll research.